workers/unity/Packages/com.improbable.gdk.core/Utility/PlayerLoopUtils.cs

```diff
@@ -0,0 +1,205 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Unity.Entities;
+using UnityEngine;
+using UnityEngine.Experimental.LowLevel;
+using UnityEngine.Experimental.PlayerLoop;
+
+namespace Improbable.Gdk.Core
+{
+    public static class PlayerLoopUtils
+    {
+        [AttributeUsage(AttributeTargets.Class, AllowMultiple = false)]
+        public class UpdateInSubSystemAttribute : Attribute
+        {
+            public UpdateInSubSystemAttribute(Type subSystemType)
+            {
+                SubSystemType = subSystemType;
+            }
+
+            public Type SubSystemType { get; }
+        }
+
+        // HACK: We need to call this specific private method to ensure the EntityDebugger shows our groups in the PlayerLoop
+        // No need to cache these methods, as each type will almost always be called once.
+        private static void InsertManagerIntoSubsystemList(PlayerLoopSystem[] subsystemList, int insertIndex,
+            ComponentSystemBase mgr, Type type)
+        {
+            var method = typeof(ScriptBehaviourUpdateOrder).GetMethod("InsertManagerIntoSubsystemList",
+                BindingFlags.Static | BindingFlags.NonPublic);
+            var genericMethod = method.MakeGenericMethod(type);
```

Comment:
are we sure this works with il2cpp - generic methods in reflection and il2cpp don't really like each other

Reply:
Will build an IL2CPP Windows client and test. Sadly we cannot do without this, so lets hope it works.

Reply:
@zeroZshadow please confirm this works?

Reply:
Works on IL2CPP Windows and Android. Jess tested on iOS, works but gave some Burst warnings.

Reply:
Jess tested this and says it did not work